inst/qml/ExploratoryFactorAnalysis.qml

```diff
@@ -49,46 +48,46 @@ Form
 	{
 		RadioButtonGroup
 		{
-			name: "factorMethod"
-			title: qsTr("Number of Factors")
-            RadioButton
-            {
-                value:      "parallelAnalysis";
-                label:      qsTr("Parallel analysis");
-                checked:    true
+			name: "factorCountMethod"
+			title: qsTr("Number of Factors based on")
+			RadioButton
+			{
+				value:      "parallelAnalysis";
+				label:      qsTr("Parallel analysis");
+				checked:    true
 
-                RadioButtonGroup
-                {
-                    name:   "parallelMethod"
+				RadioButtonGroup
+				{
+					name:   "parallelAnalysisMethod"
 					title:  ""
 
-                    RadioButton
-                    {
-                        value:      "pc"
-                        label:      qsTr("Based on PC")
-                        checked:    true
-                    }
-                    RadioButton
-                    {
-                        value: "fa"
-                        label: qsTr("Based on FA")
-                    }
-                }
+					RadioButton
+					{
+						value:		"principalComponentBased"
+						label:		qsTr("Based on PC")
+						checked:	true
+					}
+					RadioButton
+					{
+						value: "factorBased"
+						label: qsTr("Based on FA")
+					}
+				}
 				IntegerField
 				{
-					name: 			"parallelSeed"
+					name: 			"parallelSeedValue"
```

Comment:
seedValue is not necessary I think, otherwise I would add the `Analysis` to have consistency (alternatively you could also change the radiobutton value `parallelAnalysis` -> `parallel`, etc.).
```suggestion
					name: 			"parallelAnalysisSeed"
```